inventory/hosts.example

```diff
@@ -150,10 +150,13 @@ debug_level=2
 # The following options must not be used
 # - openshift_docker_options
 #openshift_docker_use_system_container=False
-# Install and run cri-o along side docker
+# Install and run cri-o
+#openshift_use_crio=False
+# If you are on atomic host, or wish to use a system container version of cri-o you can set
+# the following to True.
 # NOTE: This uses openshift_docker_systemcontainer_image_registry_override as it's override
 # just as container-engine does.
-#openshift_use_crio=False
+#openshift_crio_use_system_container=False
```

Comment:
I don't like how we're swapping openshift_use_crio = system containers by default.  I think we should do the reverse, and package should be opt-in.  This will prevent older users from breaking and also probably help preserve much of the existing logic.

Alternatively, we can replace openshift_use_crio project-wide with openshift_use_crio_system_container; add in openshift_use_crio_package where needed.

I don't think there will be a lot of overlap between the two.

Reply:
okay, I have looked over the code.  leaving 'openshift_use_crio' as-is will be okay, but I want the package variant to be opt-in, not the system container variant.  This will help us in the future, IMO, because then we will be able to check for openshift_use_crio_rpm and fail early when the option goes away.

Reply:
@mrunalp are you ok with rpm package opt in?

Reply:
@ashcrow Sure, no problem.